app/javascript/chat/draw/index.jsx

```diff
@@ -0,0 +1,169 @@
+import { h } from 'preact';
+import { useRef, useEffect, useState } from 'preact/hooks';
+import PropTypes from 'prop-types';
+import { Button } from '@crayons';
+
+function Draw({ sendCanvasImage }) {
+  const canvasRef = useRef(null);
+  const palettes = [
```

Comment:
Since these palettes do not change based on any state, they can go outside of the `Draw` component.